runtests.py

```diff
@@ -223,7 +223,8 @@ def run_test_suite(args):
         print("----------------------------------------------------------------------")
 
         from subprocess import call
-        flake_result = call(["flake8", ".", "--count"])
+        flake_result = call(["flake8", ".", "--count", "--exclude",
+                             ".venv,djstripe/migrations"])
```

Comment:
This is set in the [setup.cfg](https://github.com/kavdev/dj-stripe/blob/master/setup.cfg#L7) file. Since there aren't many migrations for the project, it isn't too much of a hassle to keep migration files up to pep8.